front/lib/api/assistant/actions/dust_app_run.ts

```diff
@@ -237,7 +237,7 @@ export class DustAppRunConfigurationServerRunner extends BaseActionConfiguration
     const params: DustAppParameters = {};
 
     for (const k of spec.inputs) {
-      if (rawInputs[k.name] && typeof rawInputs[k.name] === k.type) {
+      if (k.name in rawInputs && typeof rawInputs[k.name] === k.type) {
```

Comment:
Second condition should be enough no? If undefined it will be false

Reply:
I think you are correct.
Let's keep the first check, it's more explicit.